modules/files/files.go

```diff
@@ -24,6 +24,16 @@ func FileExistsE(path string) (bool, error) {
 	return err == nil, nil
 }
 
+// DirExistsE returns true if the given file exists
```

Comment:
```suggestion
// DirExistsE returns true if the given directory exists
```

Reply:
Actually, this method doesn't seem to check if the given `path` is a directory or not... So it seems to be identical to `FileExistsE` above?

Reply:
yeah...😅 I removed this function and used `FileExistsE` instead. At first i created it so i could return an error for a directory not existing instead of just the bool.